cmd/influx/main.go

```diff
@@ -91,9 +93,42 @@ func newApp() cli.App {
 	}
 }
 
+// This creates a new slice and replaces `-t "-FOO-TOKEN"` with `-t=-FOO-TOKEN`
+// This is necessary to do because the command line arg:
+//  `-t "-FOO-TOKEN"`` will be parsed as two separate flags instead of a flag and token value.
+func replaceTokenArg(args []string) []string {
+	newArgs := make([]string, len(args))
+	copy(newArgs, args)
+	for i, arg := range newArgs {
+		switch arg {
+		case "--token", "-t":
+			// if last element, this will be invalid later
+			if i == len(args)-1 {
+				break
+			}
+			if strings.HasPrefix(newArgs[i+1], "-") {
+				color.HiYellow("warning: %s %s interpreted as %s=%s, consider using %s=%s syntax when tokens start with a hyphen",
```

Comment:
Can you use explicit argument indexing in the format string here?

Reply:
TIL!